dev-support/submarine-dev-docker-image/build_submarine_dev_env.sh

```diff
@@ -19,47 +19,32 @@ spark_v=2.4.3
 submarine_v=0.2.0
 image_name="local/hadoop-docker:submarine"
 
+download_package() {
+  if [ -e "$1" ]; then
```

Comment:
I am not sure if `-e` ??
still is
`if [  ! -f "$1" ]; then`

Reply:
Hi, @liuxunorg 

Thanks for help to review this, `-f` is better than `-e`, I will fixed it later.

```suggestion
  if [ -f "$1" ]; then
```